draft-ietf-masque-connect-udp-listen.md

```diff
@@ -153,11 +170,103 @@ UDP Payload:
 : The unmodified UDP Payload of this proxied UDP packet (referred to as "data
 octets" in {{UDP}}).
 
+# Context ID {#contextid}
+
+Context ID values MUST always be greater than 0. A payload with value 0 MUST
+be dropped by the receiving party.
+
+An even context ID MUST be used by clients to send packets to the proxy, and an odd
+context ID MUST be used by the proxy to send packets to the client. A Client and
+a proxy MUST exhange COMPRESSION_ASSIGN packets for a given context ID to guarantee
```

Comment:
It's unclear to me what this MUST requires me to do. If this is explaining the concept, then we don't need normative text

Reply:
Removed the MUST, redundant